Symfony/CS/Fixer/Symfony/MethodArgumentDefaultValueFixer.php

```diff
@@ -22,7 +22,7 @@
 final class MethodArgumentDefaultValueFixer extends AbstractFixer
 {
     private $argumentBoundaryTokens = array('(', ',', ';', '{', '}');
-    private $variableOrTerminatorTokens = array(array(T_VARIABLE), ';', '{', '}');
+    private $variableOrTerminatorTokens = array(array(T_VARIABLE), array(T_USE), ';', '{', '}');
```

Comment:
Sth is terrible wrong with this fixer and all that neighbours. 
`variableOrTerminatorTokens` - is `T_USE` a variable? No. Is it a terminator? No, as parameters list should already be terminated by `)`, but it isn't in the list even!

What is worst, the fixer works with following props:
```php
    private $argumentBoundaryTokens = array('(');
    private $variableOrTerminatorTokens = array(array(T_VARIABLE), array(T_USE), '{', );
    private $argumentTerminatorTokens = array(',');
    private $defaultValueTokens = array('=', ')');
    private $immediateDefaultValueTokens = array('=', '{');
```

I do not say that the props should be simplified into one I just provided.
Almost all usage of `getNextTokenOfKind` method in this fixer should be replaced by `getNextMeaningfulToken` and instead of searching for tokens of given kind - which may be way outside the parameters scope (like that `T_USE`) - decide what to do based on next token.